README.rdoc

```diff
@@ -50,20 +55,39 @@ Alternatively, you can install the gem from Rubyforge:
   }
   
   response = client.track(event, properties)
-  
-  response.ok? # returns true or false
-  
-  response.http_status_code # HTTP response code, 200 is ok.
-
-  response.api_status # status field in the return body, Link to Error Codes
 
-  response.api_error_message # Error message associated with status Error Code
+  response.ok?  # returns true or false
+  response.body  # API response body
+  response.http_status_code  # HTTP response code, 200 is ok.
+  response.api_status  # status field in the return body, Link to Error Codes
+  response.api_error_message  # Error message associated with status Error Code
   
-  # Request a score forthe user with user_id 23056
+
+  # Request a score for the user with user_id 23056
   response = client.score(user_id)
-  
+
+
   # Label the user with user_id 23056 as Bad with all optional fields
-  response = client.label(user_id,{ "$is_bad" => true, "$reasons" => ["$chargeback", ], "$description" => "Chargeback issued", "$source" => "Manual Review", "$analyst" => "analyst.name@your_domain.com"})
+  response = client.label(user_id, {
+    "$is_bad" => true,
+    "$abuse_type" => "payment_abuse",
```

Comment:
Just to double-check, we are removing the $reasons field due to the addition of $abuse_type?